### PR TITLE
Standalone tool to check the status of all published pages on a site

### PR DESCRIPTION
**Pull request by geoo89**

### CODE REVIEW

File: common/check_page_status/check_page_status.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from urllib.request import Request, urlopen
+import json
+import requests
+import argparse
+import logging
+
+
+def get_sitemap(url):
+    req = Request(url + 'sitemap/?format=json')
```
Comment (sheralim012) [inline common/check_page_status/check_page_status.py:9]: is there any specific reason you aren't using `requests.get()`

File: common/check_page_status/check_page_status.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    req.add_header('Accept', 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8')
+    req.add_header('Accept-Charset', 'ISO-8859-1,utf-8;q=0.7,*;q=0.3')
+    req.add_header('Accept-Encoding', 'none')
+    req.add_header('Accept-Language', 'en-US,en;q=0.8')
+    content = json.loads(urlopen(req).read().decode('utf-8'))
+    return content
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Check status of all pages of a given IoGT site.')
```
Comment (sheralim012) [inline common/check_page_status/check_page_status.py:20]: if easier could you convert this to a management command?

**Review by sheralim012 — CHANGES_REQUESTED** [review]

File: common/check_page_status/check_page_status.py (hunk 1 above)
Comment (geoo89) [reply common/check_page_status/check_page_status.py:9]: Will do.

File: common/check_page_status/check_page_status.py (hunk 2 above)
Comment (geoo89) [reply common/check_page_status/check_page_status.py:20]: How would I do that?

**Comment by istride** [comment]
@ChrisMarsh82 I suggest closing this PR if it is no longer necessary.

**Pull request closed without merging**